extension/encoding/awslogsencodingextension/internal/constants/format.go

```diff
@@ -4,14 +4,22 @@
 package constants // import "github.com/open-telemetry/opentelemetry-collector-contrib/extension/encoding/awslogsencodingextension/internal/constants"
 
 const (
-	FormatCloudWatchLogsSubscriptionFilter = "cloudwatch_logs_subscription_filter"
-	FormatVPCFlowLog                       = "vpc_flow_log"
-	FormatS3AccessLog                      = "s3_access_log"
-	FormatWAFLog                           = "waf_log"
-	FormatCloudTrailLog                    = "cloudtrail_log"
-	FormatELBAccessLog                     = "elb_access_log"
+	FormatCloudWatchLogsSubscriptionFilter = "cloudwatch_subscription_filter"
```

Comment:
Should this one be just "cloudwatch"? That's what the PR description says

Reply:
That was the one name I was unsure of. Yes that would simplify it.